src/cpu/x64/jit_generator.hpp

```diff
@@ -1320,6 +1320,18 @@ class jit_generator : public Xbyak::CodeGenerator, public c_compatible {
         }
     }
 
+    void uni_vpblendw(const Xbyak::Xmm &x1, const Xbyak::Xmm &x2,
+                      const Xbyak::Operand &op, const int imm) {
+        assert(!x1.isZMM() && !x2.isZMM());
```

Comment:
All asserts will be removed under Release mode.
To avoid Zmm registers, I suggest the following overload:
```cpp
// NOTE: Add in future support for Zmm registers if there will be appropriate instruction
void uni_vpblendw(const Xbyak::Zmm &x1, const Xbyak::Zmm &x2,
                               const Xbyak::Operand &op, const int imm) = delete;
```

Reply:
Updated and merged into https://github.com/openvinotoolkit/oneDNN/pull/143
This one will be closed